### ISSUE TITLE
Outlook 365 calendar opening in browser

### ISSUE THREAD

**Issue by ArthurHNL** [ramboxapp/community-edition#2258]
### Steps to reproduce

1. Create an outlook 365 (for business) tab
2. Sign in
3. Click on calendar

### Expected behavior

The calendar opens inside Ranbox.

### Actual behavior

The calendar opens in my web brower (Firefox).

### ENVIRONMENT

> Rambox: v0.6.7

> OS: Windows 10 Pro 1803

> Arch: x64

**Comment by Corchoneitor** [comment]
Hi @ArthurHNL.

Outlook Calendar (or in 365) normally opens in a new window (or tab) if you are using Chrome, Firefox or other browser. That is the expected behavior in Rambox when you open try to open new "windows".

That been said, it would be a nice feature to add: when services try to open "new tabs", they should open as a tab in rambox, or a secondary Rambox window.

**Comment by maxerbox** [comment]
Yep, I will maybe add an option to enable what you mentioned. Anyway, the regexp property with an expression matching the hostname should do the trick

**Comment by ArthurHNL** [comment]
Regex property is cool and very useful, but you can't expect average users to write a regex. Unless it comes preconfigured with regexes for the default available services.

**Comment by mattiasvh** [comment]
I have the same issue on Linux platform, but the issue happens only if you enable "Try the new Outlook" (See [Try the new Outlook](https://support.office.com/en-ie/article/get-help-with-the-new-outlook-on-the-web-017014cd-2ad0-41ab-8473-6bd8c349d4f8))

When you do not enable "Try the new Outlook" this button, you can open the Outlook calendar in current Rambox tab (without opening a browser window)

**Comment by jerry1970** [comment]
Where can we find some documentation or examples for that regexp option?
